### PR TITLE
docs: update readme to correspond to latest Dataplane Selector API

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## What this PR changes/adds

updates the relevant sections in the readme files about accessing the dataplane selector API.

## Why it does that

the DPS-API was recently updated to be JSON-LD-aware

## Further notes

_List other areas of code that have changed but are not necessarily linked to the main feature. This could be method
signature changes, package declarations, bugs that were encountered and were fixed inline, etc._

## Linked Issue(s)

Closes # <-- _insert Issue number if one exists_

### CODE REVIEW

**Review by ronjaquensel — APPROVED** [review]

**Review by ndr-brt — APPROVED** [review]
we'll need to wait for the next release before merging this

**Pull request merged**
